ql/src/test/queries/clientpositive/materialized_view_parquet.q

```diff
@@ -0,0 +1,196 @@
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 2
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select deptno, name, salary, commission
+from emps_parquet_n3;
+
+explain cbo
+select emps_parquet_n3.name, emps_parquet_n3.salary, emps_parquet_n3.commission
+from emps_parquet_n3
+join depts_parquet_n2 using (deptno);
+
+select emps_parquet_n3.name, emps_parquet_n3.salary, emps_parquet_n3.commission
+from emps_parquet_n3
+join depts_parquet_n2 using (deptno);
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 3
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select empid deptno from emps_parquet_n3
+join depts_parquet_n2 using (deptno);
+
+explain cbo
+select empid deptno from emps_parquet_n3
+join depts_parquet_n2 using (deptno) where empid = 1;
+
+select empid deptno from emps_parquet_n3
+join depts_parquet_n2 using (deptno) where empid = 1;
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 4
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select * from emps_parquet_n3 where empid < 200;
+
+explain cbo
+select * from emps_parquet_n3 where empid > 120
+union all select * from emps_parquet_n3 where empid < 150;
+
+select * from emps_parquet_n3 where empid > 120
+union all select * from emps_parquet_n3 where empid < 150;
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 5 - NO MV, ALREADY UNIQUE
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select empid, deptno from emps_parquet_n3 group by empid, deptno;
+
+explain cbo
+select empid, deptno from emps_parquet_n3 group by empid, deptno;
+
+select empid, deptno from emps_parquet_n3 group by empid, deptno;
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 5 - NO MV, ALREADY UNIQUE
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select empid, name from emps_parquet_n3 group by empid, name;
+
+explain cbo
+select empid, name from emps_parquet_n3 group by empid, name;
+
+select empid, name from emps_parquet_n3 group by empid, name;
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 5
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select name, salary from emps_parquet_n3 group by name, salary;
+
+explain cbo
+select name, salary from emps_parquet_n3 group by name, salary;
+
+select name, salary from emps_parquet_n3 group by name, salary;
+
+drop materialized view mv1_parquet_n2;
+
+-- EXAMPLE 6
+create materialized view mv1_parquet_n2
+TBLPROPERTIES ('transactional'='true', 'transactional_properties'='insert_only') as
+select name, salary from emps_parquet_n3 group by name, salary;
```

Comment:
What is the difference between `EXAMPLE 5` and `EXAMPLE 6`?

Reply:
This was taken from `materialized_view_rewrite.q` contents as-is. We do not need to recreate the MV tbh but I think it's not a bit deal if we leave it this way? Lmk